2-collectors/scala-stream-collector/examples/config.hocon.sample

```diff
@@ -98,6 +98,10 @@ collector {
       # Region where the streams are located
       region = {{kinesisRegion}}
 
+      ## Endpoint url configuration to override aws kinesis endpoints,
```

Comment:
I'd say `Optional endpoint url...`, same thing below

Reply:
Done for both.